Apps/WebClient/src/ClientApp/src/components/report/ReportsComponent.vue

```diff
@@ -566,7 +566,7 @@ export default class ReportsComponent extends Vue {
         <div
             v-if="reportComponentName"
             data-testid="reportSample"
-            class="sample d-none d-md-block"
+            class="preview"
```

Comment:
Would it not have been quicker to use the `@on-is-empty-changed` event to determine if a "no records available"  message could be shown?

Immunization could have watched for records and recommendation changes before raising the event to the parent.